ClinGen/clincoded
# Connects to #669. Connects to #668. Connects to #670. Connects to #675. Select variant page and related modals

This PR contains code for the new select-variant page, the add external resource modals associated with that page (ClinVar and CAR), and parsing of data from those resources.

* The add external resource modal code has been extracted from `curator.js` to `add_external resource.js`. References have been fixed in GCI pages that previously used the add external resource modal
* The new select variant page has been created as  `select-variant.js`. Relevant CSS has been added to `_form.scss` and `_curator.scss`
* `parse-resource.js` has all the code for parsing data from ClinVar and CAR and conforming it to our schema

Testing:
* Confirm that link to select-variant page exists on dashboard
* On select-variant page, try adding a variant via the ClinVar modal and the CAR modal
* Confirm that the variant object was properly created in internal db
* On pressing 'View Variant' page, browser should be taken the `/curation-variant/?variant=[UUID]` (page does not exist yet)
* Confirm that the add ClinVar variant modal works in GCI
